src/java/org/apache/cassandra/index/sai/StorageAttachedIndex.java

```diff
@@ -200,7 +201,9 @@ public SecondaryIndexBuilder getIndexBuildTask(ColumnFamilyStore cfs,
                                                                      IndexTarget.TARGET_OPTION_NAME,
                                                                      IndexTarget.CUSTOM_INDEX_OPTION_NAME,
                                                                      IndexWriterConfig.POSTING_LIST_LVL_MIN_LEAVES,
-                                                                     IndexWriterConfig.POSTING_LIST_LVL_SKIP_OPTION);
+                                                                     IndexWriterConfig.POSTING_LIST_LVL_SKIP_OPTION,
+                                                                     LuceneAnalyzer.JSON_ANALYZER,
```

Comment:
If we have a `JSON_QUERY_ANALYZER` would it be better to call this `JSON_INDEX_ANALYZER`, or should we have all 3. So if you specify `JSON_ANALYZER` it applies to indexing and querying or you have to specify the other 2.

Reply:
I don't think it matters.